pcdscalc/be_lens_calcs.py

```diff
@@ -0,0 +1,830 @@
+'''
+Module for Beryllium Lens Calculations
+'''
+import logging
+import os
+import shutil
+from datetime import date
+from itertools import product
+
+import numpy as np
+import xraydb as xdb
+
+
+logger = logging.getLogger(__name__)
+
+# Set of Be lenses with thicknesses:
+LENS_RADII = [50e-6, 100e-6, 200e-6, 300e-6, 500e-6, 1000e-6, 1500e-6]
+
+FWHM_SIGMA_CONVERSION = 2.35482004503
```

Comment:
Should at least be a one-liner comment for these

Reply:
Ok added comments for these in 7820456 - thank you.